### PR TITLE
Core package unit tests

### PR DESCRIPTION
**Pull request by sumboid**
I've updated `packages/core/src/__tests__/model.json` to add multiple synthetic objects, please let me know if it is generated automatically: I'll update this part in such case

### CODE REVIEW

**Comment by haiodo** [comment]
model.json is generated from 

```yarn workspace @anticrm/model genmodel``` from project root from ```packages/model/src/__tests__/test_tasks.ts``` as model source file.

File: packages/core/src/__tests__/model.test.ts [hunk 1]
```diff
@@ -172,3 +173,522 @@ describe('matching', () => {
...
+
+  it('throws if domain does not exist', () => {
+    expect(() => model.getDomain('class:core.Obj' as Ref<Class<Doc>>))
+      .toThrowError()
+  })
+
+  it('returns classes', () => {
+    expect(model.getClass('class:core.Obj' as Ref<Class<Doc>>))
+      .toEqual('class:core.Obj')
+    expect(model.getClass('mixin:core.ShortID' as Ref<Class<Doc>>))
```
Comment (haiodo) [inline packages/core/src/__tests__/model.test.ts:354]: Please use: CORE_MIXIN_SHORTID

File: packages/core/src/__tests__/model.test.ts [hunk 2]
```diff
@@ -172,3 +173,522 @@ describe('matching', () => {
...
+  it('reuses proto with \'as\' method', () => {
+    const model = new Model('vdocs')
+    model.loadModel(data)
+    model.add(VDocClass)
+    model.add(refsMixin)
+
+    const shortId = 'short-id'
+    const target = {
+      ...doc1,
+      'shortId|mixin:core~References': shortId
```
Comment (haiodo) [inline packages/core/src/__tests__/model.test.ts:441]: Please use mixinKey here.

**Comment by sumboid** [comment]
Have one comment: as we decided I didn't replace domain ids, and replaced all core ids with constants, also added multiple necessary constants

**Pull request merged**